### PR TITLE
Feature/evaluation

### PR DESCRIPTION
**Pull request by UlrikeWerner**
please ignore NavBar.spec.md

### CODE REVIEW

File: src/utils/progress.spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+//import {secondToDurationData, createDurationString} from './date';
```
Comment (stefankorder) [inline src/utils/progress.spec.js:1]: Please delete out commented code.

File: src/pages/ActiveChallengePage.js [hunk 2]
```diff
@@ -39,7 +42,18 @@ export default function ActiveChallengePage() {
 				<SiteContent>
 					{challenge.length > 0 ? (
```
[the remark this answers is not in the document]
Comment (UlrikeWerner) [reply src/pages/ActiveChallengePage.js:43]: I have tested it and get truth out. In this case I have to check for the length.

[the remark this answers is not in the document]
Comment (UlrikeWerner) [reply src/pages/ActiveChallengePage.js:43]: But I can test it with find again.

File: src/pages/ActiveChallengePage.js [hunk 3]
```diff
@@ -39,7 +42,18 @@ export default function ActiveChallengePage() {
...
 						<>
-							<h1>{challenge[0]?.title}</h1>
+							<HeadLine>
+								<Icon
+									icon={challenge[0]?.logo}
```
[the remark this answers is not in the document]
Comment (UlrikeWerner) [reply src/pages/ActiveChallengePage.js:47]: I change filter to find!

File: src/pages/ActiveChallengePage.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (UlrikeWerner) [reply src/pages/ActiveChallengePage.js:43]: With find, I can prove the truthfulness! Thanks!

**Review by stefankorder — APPROVED** [review]
Good work!

**Pull request merged**